frameworks/Crystal/kemal/shard.yml

```diff
@@ -7,7 +7,7 @@ dependencies:
     version: "0.13.3"
   kemal:
     github: "sdogruyol/kemal"
-    version: "0.18.3"
+    branch: master
```

Comment:
This should point to next release containing `logging` bugfix, @sdogruyol would u be so kind? :)